packages/react-core/src/demos/DatePicker/DatePicker.md

```diff
@@ -54,23 +54,34 @@ DateRangePicker = () => {
       </SplitItem>
     </Split>
   );
-}
+};
 ```
 
 ### Date and time range picker
 
 ```js
-import { Flex, FlexItem, InputGroup, DatePicker, isValidDate, TimePicker, yyyyMMddFormat, updateDateTime } from '@patternfly/react-core';
+import {
+  Flex,
+  FlexItem,
+  InputGroup,
+  DatePicker,
+  isValidDate,
+  TimePicker,
+  yyyyMMddFormat,
+  updateDateTime
+} from '@patternfly/react-core';
 
 DateTimeRangePicker = () => {
   const [from, setFrom] = React.useState();
   const [to, setTo] = React.useState();
 
-  const toValidator = date => {
-    return isValidDate(from) && yyyyMMddFormat(date) >= yyyyMMddFormat(from) ? '' : 'To date must after from date';
+  const toValidator = (date) => {
+    return isValidDate(from) && yyyyMMddFormat(date) >= yyyyMMddFormat(from)
+      ? ''
+      : 'To date must be less than from date';
```

Comment:
I think "after" makes more sense to most people. Also it seems to be backwards, from < to.